template/plop/reduxModule/templates/redux.hbs

```diff
@@ -1,14 +1,9 @@
-import { createActions, createReducer } from 'reduxsauce';
-import Immutable from 'seamless-immutable';
+import { actionCreator, createImmutableReducer } from '../helpers';
 
-export const { Types: {{ pascalCase name }}Types, Creators: {{ pascalCase name }}Actions } = createActions(
-  {
-    noop: null, // TODO: remove this action
-  },
-  { prefix: '{{ constantCase name }}/' }
-);
+const createAction = actionCreator('{{ constantCase name }}');
 
-export const INITIAL_STATE = new Immutable({});
+export const noop = createAction('NOOP');
 
-export const reducer = createReducer(INITIAL_STATE, {
-});
+export const INITIAL_STATE = {};
+
```

Comment:
I would add here empty `HANDLERS` map like in the `locales.redux.js`

Reply:
added 👍